test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10138,6 +10139,30 @@ def forward(self, t: Tensor) -> Tuple[Tensor, Tensor]:
         x = torch.zeros(1)
         self.run_test(torch.jit.script(M()), (x,))
 
+    # This test is for shapeValueMap.
```

Comment:
this comment is pretty redundant with the name of the test; I'd remove the comment.